fiftyone/server/routes/camera.py

```
@@ -7,19 +7,174 @@
+    dataset, sample
+) -> List[_SerializedStaticTransform]:
+    """Lists static transforms for a sample."""
+    dataset_transforms: Dict[str, focam.StaticTransform] = (
+        dataset.static_transforms or {}
+    )
+    serialized_transforms: List[_SerializedStaticTransform] = []
+    seen: Set[str] = set()
+    inline_transforms: List[focam.StaticTransform] = []
+    transform_refs: List[focam.StaticTransformRef] = []
+
+    def _serialize_and_append_if_unseen(
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_serialize_and_append_if_unseen`

Add return type annotation: `None`

(ANN202)